README.md

```diff
@@ -18,6 +18,7 @@ Global:
  * `fortigate_cpu_usage_ratio`
  * `fortigate_memory_usage_ratio`
  * `fortigate_current_sessions`
+ * `fortigate_license_vdom_info`
```

Comment:
What do you think about having `fortigate_license_vdom_usage` and `fortigate_license_vdom_max` instead?
My thinking is that it makes it easier to discover, and currently it kind of breaks the [following](https://prometheus.io/docs/practices/naming/#metric-names) best-practice section:

> As a rule of thumb, either the sum() or the avg() over all dimensions of a given metric should be meaningful (though not necessarily useful). If it is not meaningful, split the data up into multiple metrics. For example, having the capacity of various queues in one metric is good, while mixing the capacity of a queue with the current number of elements in the queue is not.

Doing a `sum(fortigate_license_vdom_info)` doesn't make much sense as it contains both `max` and `usage`

Reply:
Yeah, makes more sense. 👍